pdf/pdfcanvas.cpp

```diff
@@ -453,6 +454,16 @@ QPointF PDFCanvas::fromPageToItem(int index, const QPointF &point) const
                    point.y() * page.rect.height() + page.rect.y());
 }
 
+void PDFCanvas::linksFinished(int id, const QList<QPair<QRectF, QUrl> > &links)
+{
+    PDFPage &page = d->pages[id];
+
+    page.linksAvailable = true;
```

Comment:
Just a suggestion, but would linksLoaded or something on that direction be slightly better? Available could be understood as links exist on a page.